sdk/identity/identity/src/credentials/azureCliCredential.ts

```diff
@@ -9,6 +9,21 @@ import { logger } from "../util/logging";
 
 import * as child_process from "child_process";
 
+
+function get_safe_working_dir(): string {
+    let path = process.env.PATH;
+
+    if (!path) {
+      return ".";
```

Comment:
Defaulting to `.` reintroduces the attack vector this method is intended to remove. It also seems weird to me: if the environment has no path, what are the odds we're executing from az's install directory? Do we anticipate people bundling the CLI with their applications?

Reply:
@chlowell - where should it default to?